gradle/dependencies.gradle

```diff
@@ -157,7 +157,8 @@ versions += [
   zinc: "1.9.2",
   zookeeper: "3.8.4",
   zstd: "1.5.6-3",
-  junitPlatformSuiteEngine: "1.10.2"
+  junitPlatformSuiteEngine: "1.10.2",
```

Comment:
Could you please merge those two version variables? Both of them are about junit platform, right?

Reply:
Yep, I forget they are all under junitPlatform and share same version. I will combine them.